python/mxnet/gluon/block.py

```diff
@@ -1040,7 +1061,12 @@ def register_child(self, block, name=None):
         super(HybridBlock, self).register_child(block, name)
         self._clear_cached_op()
 
-    def hybridize(self, active=True, **kwargs):
+    def hybridize(self, active=True, backend=None, backend_args=None, **kwargs):
+        self._backend = backend
+        if backend_args is None:
+            self._backend_args = {}
```

Comment:
do we really need this? isn't it default?

Reply:
Yes, we "splat" the backend_args later in the code, so has to be a dictionary or the "splat" operator (ie. the *) will error out

Reply:
I think default None is the safest option, anyway we don't need this line

we need to enforce users to pass a dictionary (since user may pass a string), so we need to add a check below before assign it to _backend_args